meta-balena-common/recipes-bsp/grub/grub_%.bbappend

```diff
@@ -0,0 +1,19 @@
+inherit deploy
+
+DEPENDS_append_class-target = " grub-conf"
+
+# we don't want grub modules in our sysroot
+# this removes them for platforms other than x86-64
+FILES_${PN}-common_remove = "${libdir}/${BPN}"
```

Comment:
Don't we need this to be removed for x86-64 also?

Reply:
@floion This comment is actually slightly incorrect. Looking at poky's `grub_2.04.bb` again:

```
FILES_${PN}-common_append_aarch64 = " \ 
    ${libdir}/${BPN} \
"
```

This only removes the grub modules from the sysroot for aarch64, because there's an exception in the original recipe to install them. For other architectures, including x86_64, these modules are contained in `grub-efi` instead of `grub-common`, so they're removed in `grub-efi_%.bbappend`.

Reply:
I've amended this comment to indicate that this only removes the modules for aarch64.